src/components/Admin/Extensions/index.js

```diff
@@ -193,10 +196,47 @@ export default class Extensions extends React.Component {
       }],
     };
 
-    const hooks = {};
+    var handleFormChanged = this.handleFormChanged.bind(this)
+    const $hooks = {
+      onAdd(instance) {
+        // console.log('-> onAdd HOOK', instance.path || 'form');
+        handleFormChanged()
+      },
+      onDel(instance) {
+        // console.log('-> onDel HOOK', instance.path || 'form');
+        handleFormChanged()
+      },
+      onSubmit(instance){
```

Comment:
we can remove this hook since it's empty

Reply:
will address in a separate pr to remove all the empty handlers.